requirements/dev.in

```diff
@@ -1,3 +1,3 @@
 -e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos[dev]
+-e git+https://github.com/kytos-ng/kytos.git@epic/vlan_pool#egg=kytos[dev]
```

Comment:
Let's leave this thread open here (that I'm opening now) just so we remember that before merging in the future to revert this once the core PR dependency lands.